test/fixtures/ledger_transaction_fixtures.js

```diff
@@ -43,6 +43,20 @@ const buildChargeEventStateWithDefaults = (opts = {}) => {
   return state
 }
 
+const buildPaymentEvents = (opts = {}) => {
+  let events = []
+  if (opts.payment_states) {
+    opts.payment_states.forEach(paymentState => {
+      let statuses = {}
```

Comment:
This is an event, so it would be better to name it this way.

This could be more readable:
```js
const event = {
    state: buildChargeEventStateWithDefaults(paymentState),
    resource_type: 'PAYMENT',
    data: {}
}
events.push(event)
```
or even:
```js
events.push({
    state: buildChargeEventStateWithDefaults(paymentState),
    resource_type: 'PAYMENT',
    data: {}
})
```